test/integration/spec/connect.js

```diff
@@ -934,13 +956,52 @@ describe('connect', function() {
       });
     });
   });
+
+  if (defaults.topology !== 'peer-to-peer') {
+    describe('track priorities', () => {
```

Comment:
@makarandp0 Nitpick: `describe('bandwidthProfile.video.maxTracks', () => {
  it('should switch off low priority tracks when the number of subscribed RemoteVideoTracks exceeds this value', () => {});
});`